### ISSUE TITLE
Landscape orientation

### ISSUE THREAD

**Issue by ale-zai** [AshRobinson/GoogleWearAlert#3]
Incorrect presentation in landscape orientation
![20 08 14 5 04 04](https://cloud.githubusercontent.com/assets/5055320/3975343/0aad59da-2806-11e4-8aa6-4ead59bcf0e9.png)

**Comment by rschiffler** [comment]
Hi @AshRobinson,

Thanks for the great library. I did a little investigation around this issue as I didn't want to stop using GoogleWearAlert only because of an orientation problem. 

A few people in forums were talking about an iOS8 bug where if the UIWindow's rootViewController property is nil, the window's frame doesn't recalculate when the device's orientation changes. I set a UIViewController on bgWindow and it seems to have fixed the problem.

If you're still happy to receive pull requests I can send you the fix, otherwise I hope this helps others.

Kindest regards,
Rose.
